### PR TITLE
JMEDQM_AdditionalPlots

### PR DESCRIPTION
**Pull request by ramankhurana**
This PR has multiple (minor) changes or fixes in the JME DQM s/w. 
- object eta vs MET Phi (Object is ele, mu, pho, jet)
- added extra status DCS bin. 
- divide endcap plots based on inside and outside tracker 
- added Alpha for dijet monitoring. 
The slides were presented in JME-DPG few weeks ago. it will be good for 2018 data taking. https://indico.cern.ch/event/697045/contributions/2863770/attachments/1586988/2509637/JME-DQM-ReReco2018.pdf

The previous PR #22167 always failed at the runTheMatrix checks. This PR is checked and tested. I will cancel that PR.

### CODE REVIEW

**Comment by jfernan2** [comment]
please test

**Comment by dmitrijus** [comment]
Please remove commented out code :)

File: DQMOffline/JetMET/interface/JetAnalyzer.h [hunk 1]
```diff
@@ -257,20 +279,20 @@ class JetAnalyzer : public DQMEDAnalyzer {
...
-  MonitorElement* mPt_EndCap_Hi;
-  MonitorElement* mPhi_EndCap_Hi;
-  MonitorElement* mConstituents_EndCap_Hi;
-  MonitorElement* mHFrac_EndCap_Hi;
-
-  MonitorElement* mPt_Forward_Hi;
-  MonitorElement* mPhi_Forward_Hi;
-  MonitorElement* mConstituents_Forward_Hi;
-  MonitorElement* mHFrac_Forward_Hi;
+//  MonitorElement* mPt_Barrel_Hi;
```
Comment (dmitrijus) [inline DQMOffline/JetMET/interface/JetAnalyzer.h]: Remove this too!

**Comment by dmitrijus** [comment]
+1

**Comment by fabiocos** [comment]
@ramankhurana several comparisons fail with empty histograms. Is this expected/desired?

**Comment by fabiocos** [comment]
@ramankhurana sorry, I am not sure to follow: what is the purpose of keeping empty histograms? They consumes memory for nothing. If an histogram is no more needed, please just remove it. Please check the output of the test 10824 above...

**Comment by ramankhurana** [comment]
Sorry, @fabiocos I missed the previous message. I should remove remaining empty histograms by end of today.

**Comment by ramankhurana** [comment]
but i am not able to see the output of the test 10824, i have tried to delete all the empty histograms in this PR, if some of them are still remaining as shown in test 10824 then I am happy to remove them. But I don't see the output of this text.

**Comment by fabiocos** [comment]
please test

**Comment by fabiocos** [comment]
@ramankhurana please check now, after a while the output of the comparison is deleted

**Comment by fabiocos** [comment]
@ramankhurana any news about your revision?

**Comment by fabiocos** [comment]
@ramankhurana any news about your revision?

**Comment by fabiocos** [comment]
@ramankhurana any news?

**Comment by fabiocos** [comment]
please test

**Comment by fabiocos** [comment]
@ramankhurana any news on the checks of empty histograms?

**Comment by fabiocos** [comment]
hold

pending reaction from the author

**Comment by dmitrijus** [comment]
-1

Author seems to be missing...

**Comment by fabiocos** [comment]
@ramankhurana if this PR is no more of interest can we close it? Otherwise can we update it?

**Comment by jfernan2** [comment]
Pinging @ramankhurana

**Comment by ramankhurana** [comment]
Hello, 

it would be better to keep it in the release, i will do the update, can you please tell me what is the target release for which I should do this update.

**Comment by jfernan2** [comment]
10_3_X is the current master release, and suits fine these Offline DQM changes

**Comment by fabiocos** [comment]
@jfernan2 @ramankhurana @andrius @schneiml what do we do with this PR? It looks like there is no activity since months, apparently it is not needed....

**Comment by andrius** [comment]
Hi @fabiocos, please contact right andrius

**Comment by fabiocos** [comment]
@andrius-k please see the message above

**Comment by andrius-k** [comment]
@ramankhurana what is the status with the empty histograms mentioned by @fabiocos?

**Comment by andrius-k** [comment]
@ramankhurana are there any news regarding this PR? If not, I think, it could be closed until you'll get back to it.

**Comment by fabiocos** [comment]
@ramankhurana @andrius-k if there is no further reaction/activity I will close this PR at the next ORP, it may be resumed when really needed

**Comment by fabiocos** [comment]
@ramankhurana I close this PR as it is inactive and without reaction on your side since long time. In case it is really needed we may reopen it or have a new one with updated code.

**Pull request closed without merging**